src/components/common/Button/Button.module.scss

```diff
@@ -10,11 +10,13 @@
   line-height: 24px;
   font-weight: 600;
   padding: 10px 30px;
+  @extend %button-hover-before;
+  // background-color: #2a2a2a;
 
   &:not(.noHover):hover {
-    color: $color-light-primary !important;
-    background-color: $primary;
-    text-decoration: none;
+    // color: #ffffff !important;
```

Comment:
Jeżeli linijki kodu są zbędne to można je usunąć w sumie

Reply:
jesne, usunięte